libs/application-generic/src/services/auth/community.auth.service.ts

```diff
@@ -350,12 +350,6 @@ export class CommunityAuthService implements IAuthService {
     );
   }
 
-  @CachedEntity({
-    builder: ({ apiKey }: { apiKey: string }) =>
-      buildAuthServiceKey({
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** There's a few more occurrences of `buildAuthServiceKey` (including the cache invalidation for the key) that we need to remove from the code - [ref](https://github.com/search?q=repo%3Anovuhq%2Fnovu%20buildAuthServiceKey&type=code)‏